### DISCUSSION TITLE
Hit lighting fade way too fast while using classic mod with "Fade out hit circles earier" enabled

### DISCUSSION THREAD

**Discussion by DarkbitNetwork** [ppy/osu#23442] [Q&A] [answered]
I have noticed that hit lighting fades out abnormally quick at the time when the option was available but I didn't bother asking until now.

I know the hit circle is supposed to fade out earlier with the option enabled which obviously includes the hit lighting but I think this is way too fast. It's as if the hit lighting last only for 1 frame before instantly disappearing.

**Is this intended to be the case?**
### Video: 
Recorded with OBS Studio 29.0.2 (64 bit)

https://user-images.githubusercontent.com/43719333/236835591-8a8fefa7-e7d0-4c35-bf42-b029f6ac036d.mp4

**Comment by peppy** [comment]
This is an implementation detail of the argon skin – hit lighting is actually inside the hit circle here while it isn't on the classic skins. Will need some consideration to make that work.

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/issues/23771
